polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -281,7 +268,15 @@ $disabled-fade: 0.6;
   word-break: break-all;
 }
 
+.SecondaryActions {
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
```

Comment:
```suggestion
  // stylelint-disable-next-line
```

Remove the comment that it was added by the migrator since this wasn't.

Reply:
At this point we are just trying to unblock adding the linter despite all the warnings. Eventually we would expect they will be cleaned up though I don't know that it will be done automagically because they are quite context specific.